statistics-dashboard-service/src/main/java/eu/europeana/statistics/dashboard/service/persistence/Field.java

```diff
@@ -0,0 +1,33 @@
+package eu.europeana.statistics.dashboard.service.persistence;
+
+/**
+ * This enum contains the different fields that can be queried.
+ */
+public enum Field {
```

Comment:
There is a slight difference in that the `Field` class is meant to be a list of the possible Mongo query fields. This is not the same as the `FacetValue` (because not all fields are possible facets). This is why I thought it was better to keep them separate. One way to make this nice is to give each `FacetValue` a second attribute of type `Field` so that the connection can be made easily. What do you think?

Reply:
We can leave as it is then